siriuspy/siriuspy/search/id_search.py

```diff
@@ -18,23 +22,77 @@ class IDSearch:
         'CATERETE': 'SI-07SP:ID-APU22',  # titular: VPU19
         'EMA':      'SI-08SB:ID-APU22',  # titular: IVU18
         'MANACA':   'SI-09SA:ID-APU22',  # titular: 2 x APU22
-        'SABIA':    'SI-10SB:ID-EPU50',  # titular: 2 x DELTA52
+        # 'SABIA':    'SI-10SB:ID-EPU50',  # titular: 2 x DELTA52
+        'SABIA':    'SI-10SB:ID-DELTA52',  # titular: 2 x DELTA52
         'IPE':      'SI-11SP:ID-APU58',  # titular: 2 x APPLE-II
         'PAINEIRA': 'SI-14SB:ID-WIG180',  # titular: IVU18
         'SAPUCAIA': 'SI-17SA:ID-PAPU50',  # titular: 2 x APU22
     }
 
     _idname2beamline = {v: k for k, v in _beamline2idname.items()}
 
+    _idparam_fields = (
+        'PERIOD_LENGTH',  # [mm]
+        'KPARAM_MIN',  # [mm]
+        'KPARAM_MAX',  # [mm]
+        'KPARAM_PARKED',  # [mm]
+        'KPARAM_POL_CHANGE',  # [mm]
+        'KPARAM_TOL',  # [mm]
+        'PPARAM_MIN',  # [mm]
+        'PPARAM_MAX',  # [mm]
+        'PPARAM_PARKED',  # [mm]
+        'PPARAM_TOL',  # [mm]
+        )
+
+    _idname2params = {
+        # NOTE: for EPU50 there is a large discrepancy
```

Comment:
why is this note here and not in a more appropriate place?

Reply:
because this parameter dict contains the primary source of tolerance values for KParam and PParam within our python packages.

Reply:
elaborating on this issue: the appropriate primary source of such parameter/data, in my view, is a table in the control system constant service. the epics base IOC that ARO is developing could get it from csconsts. but this would have to be communicated and implemented (i think this mechanism for epics base IOC would be new). maybe we can talk to ARO and implement this change in the future, if we all agree.